File: SaasFeeGuides/SaasFeeGuides.ViewModels/ActivitySkuPrice.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SaasFeeGuides.ViewModels
{
    public class ActivitySkuPrice
    {
        public int ActivitySkuId { get; set; }
        public string Name { get; set; }
        public IList<Content> DescriptionContent { get; set; }
        public string DiscountCode { get; set; }
        public double Price { get; set; }
        public int MaxPersons { get; set; }
        public int MinPersons { get; set; }
        public double? DiscountPercentage { get; set; }
        public DateTime? ValidFrom { get; set; }
        public DateTime? ValidTo { get; set; }
    }
}

File: SaasFeeGuides/SaasFeeGuides/ClientApp/src/app/viewModels/activitySkuPrice.ts

import { Content } from './content';


export class ActivitySkuPrice {

  constructor(
    public activitySkuId: number,
    public name: string,
    public descriptionContent: Content[],
    public discountCode: string,
    public price: number,
    public maxPersons: number,
    public minPersons: number,
    public discountPercentage :number,
    public validFrom :Date,
    public validTo: Date
  ) {
  }

}


